app/grandchallenge/reader_studies/management/commands/migrate_to_display_sets.py

```diff
@@ -29,9 +29,9 @@ def handle(self, *args, **options):
                 )
                 continue
 
-            view_content = {"main": "generic-medical-image"}
+            view_content = {"main": ["generic-medical-image"]}
             if "main-overlay" in content_keys:
-                view_content["main-overlay"] = "generic-overlay"
+                view_content["main"].append("generic-overlay")
```

Comment:
This view content never actually gets saved.

Reply:
It does not need to be saved, it's a variable that gets passed to the `migrate_reader_study_to_display_sets` method in utils.

Reply:
Does CIRRUS handle this correctly then when it is unset on the reader study?

Reply:
Should be ok. From the [documentation](https://grand-challenge.org/documentation/viewer-layout/):

> No view content, no hanging protocol. In this case, the viewer will use a default hanging protocol with a single view. The first image will be shown, optionally with an overlay if one is present. 

It might still be a good idea to set it explicitly. I can write a management command for that and create a separate pr.

Reply:
OK - we should check that CIRRUS shows an error if an image cannot be displayed in this configuration.